en/components/dock-manager.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Manage Angular Page Layout | Dock Manager | Ignite UI | Infragistics
```

Comment:
```suggestion
title: Dock Manager
```

Reply:
This title is generated following our SEO process and should remain the same.

Reply:
Are you sure because other topics in this repo just use the component name?

Reply:
Yes, but not all topics follow the SEO guidelines currently. Look at the grid ones for example, they follow the guidelines.